SC_redesign/Crop_and_Soil/soil/carbon_cycling/residue_partition.py

```diff
@@ -160,3 +162,142 @@ def _determine_plant_metabolic_active_carbon_usage(decomposition_moisture_effect
         """
         return decomposition_moisture_effect * decomposition_temperature_effect * \
             plant_metabolic_carbon_amount * metabolic_active_carbon_rate
+
+    @staticmethod
+    def _determine_plant_metabolic_to_soil_carbon_amount(plant_metabolic_carbon_amount: float,
+                                                         tillage_fraction: float) -> float:
+        """This method calculates the the amount of metabolic carbon incorporated into soil during tillage (kg/ha)
+
+        Parameters
+        ----------
+        plant_metabolic_carbon_amount: float
+            amount of metabolic carbon in plant (kg/ha)
+        tillage_fraction: float
+            Fraction of metabolic carbon incorporated into soil during tillage (unitless)
+        Returns
+        -------
+        float
+            the amount of metabolic carbon incorporated into soil during tillage (kg/ha)
+
+        References
+        -------
+        pseudocode_soil S.6.B.I.6
+        """
+        return plant_metabolic_carbon_amount * tillage_fraction
+
+    @staticmethod
+    def _determine_plant_structural_to_slow_or_active_rate(plant_residue_metabolic_fraction: float,
+                                                           structural_decomposition_factor=0.076) -> float:
+        # TODO: check with subject expert for structural_decomposition_factor's default value. issue #428
+        """This method calculates the rate at which above ground structural carbon decomposes into slow or active carbon
+
+        Parameters
+        ----------
+        structural_decomposition_factor: float, default = 0.076
+            structural decomposition factor (unitless)
+        plant_residue_metabolic_fraction: float
+            fraction of plant residue that is metabolic (unitless)
+        Returns
+        -------
+        float
+            the rate at which above ground structural carbon decomposes into slow or active carbon (unitless)
+
+        References
+        -------
+        pseudocode_soil S.6.B.I.9
+        """
+        # TODO: contradiction with the equation in pseudocode_soil. issue #427
+        return structural_decomposition_factor * math.exp(-3) * 1 - plant_residue_metabolic_fraction
```

Comment:
```suggestion
        return structural_decomposition_factor * math.exp(-3) * (1 - plant_residue_metabolic_fraction)
```
Pretty sure parentheses are needed here, otherwise it just multiplies by 1 and then subtracts `plant_residue_metabolic_fraction` which (I think) is not the fraction that should be used here.